VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.hpp

```diff
@@ -204,6 +205,8 @@ class EXPORT VoodooI2CDeviceNub : public IOService {
     bool has_apic_interrupts {false};
     bool has_gpio_interrupts {false};
     bool use_10bit_addressing {false};
+    bool force_polling { false };
+    IOPCIDevice *pci_device { nullptr };
```

Comment:
Maybe we can just make them local variables since they are not used elsewhere.

Reply:
Yeah. Sounds right.